### ISSUE TITLE
Possible to add option for clockwise/countclockwise in main page?

### ISSUE THREAD

**Issue by olskar** [nidayand/motor-on-roller-blind-ws#5]
Hello!

Nice project!

Is it possible to add option for clockwise/countclockwise in main page? As of now I need to know what way it should turn before I have access to the main interface.

**Comment by nidayand** [comment]
Hi,
I was thinking about that... but I didn't have the energy to create another HTML page. Sorry, don't have the time to do it right now though so I'll keep it as an enhancement request

**Comment by petrepa** [comment]
Is there anyhow a way to change the rotation? How can I possibly do a hard reset of a configured unit?

**Comment by nidayand** [comment]
yeah, I know, it isn't perfect... What you can do is to uncomment row 272 and 273 and flash it via cable and then revert back and flash again.. (or use esptool.py to clean it completely)

  //clean FS, for testing
  //helper.resetsettings(wifiManager);
